### ISSUE TITLE
feature(?) request. Non-daily notes. 

### ISSUE THREAD

**Issue by Incblob** [pyrochlore/obsidian-tracker#81]
Hi, since graphs like pi chart don't need a date, is it possible to allow arbitrary notes to be scanned?
You don't even need dates for other graphs as long as you use for example the note name or some other #xvalue for example.

I'm trying to create a dashboard of sorts, where I keep track of progress in notes, then aggregate the results using tracker. However since the notes aren't daily notes, I can't use them even if I don't need the date. 

Don't know how easy or difficult it would be to implement, but at least bullet and piechart don't use dates. dvFields etc. could be used more easily without restricting to daily notes.

Thanks for your time.

**Comment by pyrochlore** [comment]
I’ll be here if there are more questions.

**Comment by pyrochlore** [comment]
I will close this issue. If the problems remain, you are welcome to reopen it.

**Comment by Gnopps** [comment]
I'm trying to capture completed dates from my tasks that are not present in my daily notes. I've looked at the xDataset examples but haven't been able to figure it out. How do I capture the date from the below string using text search?

Date to capture:
"✅ 2022-01-1"

Would it possible with some advice here please?

**Comment by EdJoJob** [comment]
@Gnopps 
From what I have been able to work out, you will need to have a regex that has a `value` named group (i.e. `(?<value>...)` which will need sub-groups for year, month and day.  So for you it would look like `:white_check_mark:\s(?<value>([0-9][4])-([0-9][2])-([0-9][1,2]))` and ensuring that wither your yaml block includes `dateformat: YYYY-MM-D`, or your default date format is `YYYY-MM-D` in the plugin settings

**Comment by Gnopps** [comment]
Thanks for your reply @EdJoJob but I still haven't been able to get it working.

What I'm looking to do is to get a count of occurrences per date displayed. The date is captured as I specified above "✅ 2022-01-01" (I made mistake, should be YYYY-MM-DD not YYYY-MM-D) and the count is how many occurrences of the date is found like this.

I've been experimenting trying to get this work but am not really getting anywhere.
